packages/ra-ui-materialui/src/input/FileInputPreview.tsx

```diff
@@ -30,7 +38,7 @@ export const FileInputPreview = (props: FileInputPreviewProps) => {
                 title={translate('ra.action.delete')}
                 size="small"
             >
-                <RemoveCircle className={FileInputPreviewClasses.removeIcon} />
+                {removeIcon}
```

Comment:
I'm afraid you're no longer passing the `className` here, which is a BC. You should pass it to whatever React Node `removeIcon` is.

Reply:
Thanks for pointing this out. There are different ways to fix this and I now pushed a version where the icon is handed in as a component and not a react element. I hope this is fine for you.